library/Translate/test/Desugar/Sexp.hs

```diff
@@ -0,0 +1,254 @@
+module Desugar.Sexp (top) where
+
+import qualified Juvix.Desugar.Passes as Desugar
```

Comment:
OK, so here we're testing these functions?

Reply:
Yes